app/docs/how-to/streamline-curbside.md

```diff
@@ -0,0 +1,255 @@
+---
+title: "Streamline curbside pickup with Routegy and Trello | How-to"
+description: "How do I streamline curbside pickup with Routegy and Trello?"
+sidebar: false
+---
+
+# Streamline curbside pickup with Routegy and Trello
+
+This document describes how to streamline your curbside delivery process using [Routegy](https://routegy.com) and [Trello](https://trello.com). 
+
+Customers simply pull up, scan a QR code placed on a strofront or a curbside sign, and fill a simple form to let you know that they are ready to pickup their order.
```

Comment:
> Customers pull up, scan a QR code placed on your restaurant signage, and fill out a simple form to let you know that they are here.

Tried to cut it down a bit. Removed double usage of "simple". Removed the "or" in there as well.

Reply:
Reworded it to:

> Customers pull up, scan a QR code placed on a storefront or curbside, and the staff inside inside are notified.